### PR TITLE
added compatibility with Bubble_StockMovements

### PR DESCRIPTION
**Pull request by sprankhub**
https://github.com/jreinke/magento-stock-movements

### CODE REVIEW

File: src/app/code/community/Shopgate/Framework/Model/Cataloginventory/Stock.php [hunk 1]
```diff
@@ -19,7 +19,12 @@
  * @copyright Shopgate Inc
  * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License, Version 2.0
  */
-class Shopgate_Framework_Model_Cataloginventory_Stock extends Mage_CatalogInventory_Model_Stock
+if (Mage::helper('core')->isModuleEnabled('Bubble_StockMovements')) {
```
Comment (augsteyer) [inline src/app/code/community/Shopgate/Framework/Model/Cataloginventory/Stock.php]: Please use ```Mage::getConfig()->getModuleConfig(‘ Bubble_StockMovements’)->is(‘active’, ‘true’)``` as isModuleEnabled is not supported in versions CE 1.4.0.0, 1.4.0.1

**Review by augsteyer — CHANGES_REQUESTED** [review]

**Comment by sprankhub** [comment]
@augsteyer please do not support Magento 1.4 :D Anyway, fixed.

**Comment by augsteyer** [comment]
that's funny, that's why pushing this fix is going so long. We are considering dropping 1.4 support, then your modifications will not be needed anymore as this class rewrite is for 1.4 specifically :)

**Comment by sprankhub** [comment]
What is the current status here?

**Review by Shopgate-Travis — APPROVED** [review]
Sorry for the delay. My colleague is on vacation and we didn't hand this over.

Thank you for your contribution!

**Pull request merged**